x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/general.ts

```diff
@@ -279,22 +289,6 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
   });
 
-  async function createConnector(connectorName: string) {
```

Comment:
Moved these to the utils.ts file. Maybe we should move them to the service api like cases 🤷‍♂️